llabake/Hello-Books
# Ft add book react redux 154038634

#### What does this PR do?
This PR adds book component
#### Description of Task to be completed?
Admin should be able to add book to the library
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* visit the port where the webpack-dev-server started
* on successful login, on navigation to the add book page, you should be able to add a book
#### What are the relevant pivotal tracker stories?
154038634
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/34623938-909f957e-f253-11e7-9eb9-08c3f134f203.png)
